lms/djangoapps/mobile_api/course_info/tests.py

```diff
@@ -9,35 +9,40 @@
 
 from courseware.tests.factories import UserFactory
 from xmodule.html_module import CourseInfoModule
+from xmodule.modulestore import ModuleStoreEnum
 from xmodule.modulestore.django import modulestore
 from xmodule.modulestore.tests.factories import CourseFactory
-from xmodule.modulestore.tests.django_utils import ModuleStoreTestCase
 from xmodule.modulestore.xml_importer import import_from_xml
 
+from ..tests import (
+    MobileCourseAPITestCase, MobileCourseAccessTestMixin, MobileEnrolledCourseAccessTestMixin, MobileAuthTestMixin
+)
 
-class TestCourseInfo(APITestCase):
+
+class TestCourseInfo(MobileCourseAPITestCase):
     """
-    Tests for /api/mobile/v0.5/course_info/...
+    Base test class for tests for /api/mobile/v0.5/course_info/...
     """
-    def setUp(self):
```

Comment:
No.  It's so all subclasses can share a common hierarchy and implicitly inherit from `MobileCourseAPITestCase`.
Also, I was thinking there may be common `setUp` required, but eventually didn't need it.